sanity-io/structured-content-2022
# Nav micro animations

# Nav micro animations

## Intent

- Nav slide in/out animation on frontpage
- Menu links slide in/out on small viewports

Fixes [this](https://app.shortcut.com/sanity-io/story/15474/implement-micro-animations) Shortcut Story with input and approval from Sindre.

## Details

(Was told to disregard the "Buttons" item in the associated Story.)

This PR also fixes #23. 

## Testing this PR

1. Open the Preview Deploy for this branch
2. Scroll down to where the Nav header usually would appear and verify that it slides in
3. Scroll back up and verify that it slides out again instead of disappearing
4. Open "Responsive Design Mode" and shrink the viewport until the "Menu" button appears
5. Click the button to open the menu and verify that the items are animated in one-by-one, and that closing and opening the menu again consistently triggers the same animation
